ros2launch/ros2launch/api/api.py

```diff
@@ -36,6 +40,32 @@ def __init__(self, msg, paths):
         self.paths = paths
 
 
+class NoKeystoreProvidedError(Exception):
+    """Exception raised when a keystore is not provided."""
+
+    def __init__(self):
+        super().__init__(('--no-create-keystore was specified and '
+                          'no keystore was provided'))
+
+
+class NonexistantKeystoreError(Exception):
+    """Exception raised when keystore is not on disk."""
+
+    def __init__(self, keystore_path):
+        super().__init__(('--no-create-keystore was specified and '
+                          f'ROS_SECURITY_KEYSTORE="{keystore_path}" '
+                          'does not exist'))
+
+
+class InvalidKeystoreError(Exception):
+    """Exception raised when provided keystore isn't valid."""
+
+    def __init__(self, keystore_path):
+        super().__init__(('--no-create-keystore was specified and '
+                          f'ROS_SECURITY_KEYSTORE="{keystore_path}" '
```

Comment:
Here as well, these variables should probably not be in errors because we're not using them anymore.

Reply:
c40721f